mlflow/server/js/src/components/MetricsPlotView.js

```diff
@@ -107,12 +111,13 @@ export class MetricsPlotView extends React.Component {
   };
 
   render() {
+    const { onClick, onRelayout } = this.props;
     const plotProps =
       this.props.chartType === CHART_TYPE_BAR
         ? this.getPlotPropsForBarChart()
         : this.getPlotPropsForLineChart();
     return (
-      <div className='metrics-plot-view-container'>
+      <div className="metrics-plot-view-container">
```

Comment:
single-quote in all places. Linter: [tinyurl.com/mlflowjs](https://tinyurl.com/mlflowjs)

Reply:
Will fix this.